.github/workflows/python-package.yml

```diff
@@ -17,14 +17,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, "3.10", "3.11"]
```

Comment:
For `case-utils`, we've considered it overkill to test against each version between the oldest and newest.  I appreciate it might different for this package.  @red-ship-it , is there a good reason we should test `ExifRead` against each Python version?